recipes/causalml/meta.yaml

```diff
@@ -0,0 +1,85 @@
+{% set name = "causalml" %}
+{% set version = "0.12.3" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/causalml-{{ version }}.tar.gz
+  sha256: 4d85afa0590e4c60818311c2be9a5be64b8c6964b134c77654b738df551454fa
+
+build:
+  number: 0
+  skip: true   # [win]
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - pip
+    - pytest
```

Comment:
pytest doesn't seem to appear in the setup.py file. I don't think you need it in this section.